### DISCUSSION TITLE
SQLAlchemy generic and native enum

### DISCUSSION THREAD

**Discussion by francipvb** [sqlalchemy/sqlalchemy#11785] [Usage Questions]
Hello all,

I thought that the generic enum type is a sort of **all-in-one** type. However, I found that, for example, the `create_type` is not provided to the native type when the generic enum is applied to a column. Is this an expected behavior?

Thank you.

**Comment by CaselIT** [comment]
Hi,

this seems a duplicate of https://github.com/sqlalchemy/sqlalchemy/issues/10604. please continue there
